setup.py

```diff
@@ -17,8 +17,8 @@
         "console_scripts": ["tmsc=tmsc.__main__:main"],
     },
     keywords=["machine learning on source code", "topic modeling",
-              "github", "bblfsh", "babelfish", "ast2vec"],
-    install_requires=["ast2vec>=0.3.8-alpha"],
+              "github", "bblfsh", "babelfish"],
+    install_requires=["sourced-ml>=0.5.1", "ast2vec>=0.3.8-alpha"],
```

Comment:
ast2vec is dead

Reply:
I know, was kept just as a hack to satisfy internal imports 😕